android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -56,8 +57,8 @@ bool CreateDecoderAndParse(AvifDecoderWrapper* const decoder,
   decoder->decoder->ignoreXMP = AVIF_TRUE;
   decoder->decoder->ignoreExif = AVIF_TRUE;
 
-  // Turn off 'clap' (clean aperture) property validation. The JNI wrapper
-  // ignores the 'clap' property.
+  // Turn off libavif's 'clap' (clean aperture) property validation. The
+  // property is validated by the JNI wrapper.
```

Comment:
Nit: I just realized we can say more: by validating the clap property ourselves we can ignore an invalid clap property (rather than fail).

Reply:
Done.